### PR TITLE
Update whitelist to allowlist

### PR DESCRIPTION
**Pull request by chybie**
What's changed, or what was fixed?
- update whitelist to allowlist

**Fixes:** N/A

**Target Live Date:** YYYY-MM-DD

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/fundamentals/security/csp/index.md [hunk 1]
```diff
@@ -55,10 +55,10 @@ probably isn't. The browser happily downloads and executes any code a page
 requests, regardless of source.
 
 Instead of blindly trusting _everything_ that a server delivers, CSP defines the
-`Content-Security-Policy` HTTP header, which allows you to create a whitelist of
+`Content-Security-Policy` HTTP header, which allows you to create a allowlist of
```
Comment (OrKoN) [inline src/content/en/fundamentals/security/csp/index.md:58]: `an allowlist`?

File: src/content/en/fundamentals/security/csp/index.md [hunk 2]
```diff
@@ -232,9 +232,9 @@ This can't be used for `frame-ancestors`, `report-uri`, or `sandbox`.
 
 ## Inline code is considered harmful
 
-It should be clear that CSP is based on whitelisting origins, as that's an
+It should be clear that CSP is based on allowlisting origins, as that's an
 unambiguous way of instructing the browser to treat specific sets of resources
-as acceptable and to reject the rest. Origin-based whitelisting doesn't,
+as acceptable and to reject the rest. Origin-based allowlisting doesn't,
```
Comment (jackfranklin) [inline src/content/en/fundamentals/security/csp/index.md:237]: "Using an origin-based allowlist" perhaps?

File: src/content/en/fundamentals/security/csp/index.md [hunk 3]
```diff
@@ -232,9 +232,9 @@ This can't be used for `frame-ancestors`, `report-uri`, or `sandbox`.
 
 ## Inline code is considered harmful
 
-It should be clear that CSP is based on whitelisting origins, as that's an
+It should be clear that CSP is based on allowlisting origins, as that's an
```
Comment (jackfranklin) [inline src/content/en/fundamentals/security/csp/index.md:235]: "an allowlist of origins" ?

File: src/content/en/fundamentals/security/csp/index.md [hunk 4]
```diff
@@ -232,9 +232,9 @@ This can't be used for `frame-ancestors`, `report-uri`, or `sandbox`.
 
 ## Inline code is considered harmful
 
-It should be clear that CSP is based on whitelisting origins, as that's an
+It should be clear that CSP is based on allowlist origins, as that's an
 unambiguous way of instructing the browser to treat specific sets of resources
-as acceptable and to reject the rest. Origin-based whitelisting doesn't,
+as acceptable and to reject the rest. Origin-based allowlist doesn't,
```
Comment (mathiasbynens) [inline src/content/en/fundamentals/security/csp/index.md:237]:
```suggestion
as acceptable and to reject the rest. Origin-based allowlists don't,
```

File: src/content/en/fundamentals/security/csp/index.md [hunk 5]
```diff
@@ -301,7 +301,7 @@ out-of-line, but that's a tradeoff that's well worth making.
 ### If you absolutely must use it ...
 
 CSP Level 2 offers backward compatibility for inline scripts by allowing you to
-whitelist specific inline scripts using either a cryptographic nonce (number
+allowlist specific inline scripts using either a cryptographic nonce (number
```
Comment (mathiasbynens) [inline src/content/en/fundamentals/security/csp/index.md:304]: "allowing you to allowlist" sounds awkward. Maybe "by letting you allowlist" or "by letting you add specific inline scripts ... to the allowlist"?

Comment (chybie) [reply src/content/en/fundamentals/security/csp/index.md]: I change it to "by allowing you to add specific inline scripts to the allowlist"

Comment (mathiasbynens) [reply src/content/en/fundamentals/security/csp/index.md]: Hmm, the repetition of "allow" is what's awkward IMHO.

**Pull request merged**